File: projects/m1/032-taxi-fare/solutions/enricozoboli/solution_taxi_fare.py
def taxi_fare(distance):
    """
    This function take a distance parameter in kilometers
    and return the total fare in Euros.
    """
    total_fare = (distance / 140) * 0.25 + 4.00
    return f"Total fare: €{total_fare:.2f}"

def main():
    print(taxi_fare(3400))  
    print(taxi_fare(89000))
    print(taxi_fare(280))

if __name__ == "__main__":
    main()

File: projects/m1/032-taxi-fare/solutions/valentinaBazzacchi/solution-taxi-fare.js
function taxiFare(kilometers) {
    let baseFare = 4.00;
    let plusFare = 0.25;
    plusFare = (kilometers / 0.14 ) * 0.25;
    let totalFare = (plusFare + baseFare).toFixed(2);
    return alert("The total fare is: € " + totalFare); 
    }
         
    let onlyNumbers = /^[0-9\.]+$/;
    let distance = prompt("Enter the distance travelled in kilometers");
    
    if (!onlyNumbers.test(distance)) {
        alert("Invalid Entry!");
    } else {
        taxiFare(distance);
    }